docs/getting-started/function-triggers.md

```diff
@@ -0,0 +1,130 @@
+# Triggers, Brokers, and Event Sources
+
+In the first Functions tutorial, you created a basic function that could receive
+and respond to HTTP requests. Now, let's create a new function that can respond
+to CloudEvents that are sent by Knative Eventing resources. In this tutorial, you
+will create a simple Knative Event Source to generate CloudEvents, a Knative Broker
+to send and receive these events, and a Knative Trigger to connect the event source
+to your Knative Function. First, create and deploy the function.
+## Create a CloudEvent Function
+
+To create and deploy a CloudEvent function that echoes the CloudEvent it receives,
+you can use the `kn func create` and `kn func deploy` commands.
+
+Create the function by running the command:
+
+```{ .console }
+func create -l go -t cloudevents echo
```

Comment:
```suggestion
func create -l go -t cloudevents echo
cd echo
```
This avoids the user from needing to use `-p echo` in the next step, and if they are like me they can `code .` and work on IDE :-)